mempool/src/tests/shared_mempool_test.rs

```diff
@@ -53,9 +52,14 @@ fn test_consensus_events_rejected_txns() {
 }
 
 #[test]
-fn test_state_sync_events_committed_txns() {
-    let (mut state_sync_sender, state_sync_events) = mpsc::channel(1_024);
-    let smp = MockSharedMempool::new(Some(state_sync_events));
+fn test_mempool_notify_committed_txns() {
+    // Create runtime for the mempool notifier and listener
+    let runtime = Builder::new_multi_thread().enable_all().build().unwrap();
+    let _enter = runtime.enter();
```

Comment:
This is like always the weirdest thing to me, and I swear someday will confuse us all.

Reply:
Yeah, gotta love tokio 😄